### ISSUE TITLE
Add ability to categorise entries without meaning

### ISSUE THREAD

**Issue by dadepo** [Yorubaname/yorubaname-website#24]
As a user of the YorubaName website, I want a section where name entries with no meanings can be quickly viewed, so I can easily find entries I can help with by providing the missing meanings.

**Comment by kolatubosun** [comment]
Great idea!

**Comment by dadepo** [comment]
From the discussion in the issue https://github.com/Yorubaname/yorubaname-website/issues/28, a question arose: 

> what marks something as "incomplete"? For me, even if a name doesn't have "famous people" or "variant", it can still be complete. How do we train the app to know which names to put in the "incomplete" group?

This question is important as we need a way to know when an entry should be regarded as incomplete and thus should be displayed on the website appropriately. In other to be able to do this, the plan would be to make use of the completion indicator requested in the issue https://github.com/Yorubaname/yorubaname-website/issues/3

Basically we would have a completion threshold, as defined by the completion indicator, and if an entry is lower than this threshold, it would automatically be displayed in the website as an entry that needs attention.

**Comment by kolatubosun** [comment]
And going with what is planned via #34, it might just be easy to tie the checkbox to a button either on the dashboard or on the front end where one can click and it shows all the entries with the "incomplete" or "meaning unknown" checkbox.
